ddl/ddl.go

```diff
@@ -75,6 +75,8 @@ type ddl struct {
 	uuid      string
 	jobCh     chan struct{}
 	jobDoneCh chan struct{}
+	// drop database/table task
+	taskCh chan struct{}
```

Comment:
If the chan is only used for drop task. So it is better to rename it to dropTaskCh.